operations/host_spawn.go

```diff
@@ -353,14 +355,14 @@ func getJasperCommands(projectRef *model.ProjectRef, directory string, quiet, dr
 			return nil, errors.Wrapf(err, "error making directory '%s'", dir)
 		}
 
-		commandNubmer := idx // to avoid logging a stale number
+		commandNumber := idx + 1 // to avoid logging a stale number
 		cmd := jasper.NewCommand().Directory(dir).Add([]string{obj.Command}).
 			SetErrorSender(level.Error, grip.GetSender()).
 			Prerequisite(func() bool {
 				grip.Info(message.Fields{
 					"directory": dir,
 					"command":   obj.Command,
-					"index":     commandNubmer,
```

Comment:
In this case I'm not sure you should put idx + 1 because the field name "index" does imply actual indexing. Could change to "command_number" or remove the +1